kotlin/src/test/java/org/apache/zeppelin/kotlin/KotlinInterpreterTest.java

```diff
@@ -67,7 +96,6 @@ public void testIncomplete() throws Exception {
   public void testCompileError() throws Exception {
     InterpreterResult result = interpreter.interpret("prinln(1)", context);
     assertEquals(ERROR, result.code());
-    assertEquals(
-        "Unresolved reference: prinln", result.message().get(0).getData().trim());
+    assertEquals("Unresolved reference: prinln", result.message().get(0).getData().trim());
   }
 }
```

Comment:
Add testCancel() and testRuntimeError() to check stderr is handled correctly

Reply:
Add KotlinSparkInterpreterTest to check spark related stuff

Reply:
done

Reply:
> Add testCancel() and testRuntimeError() to check stderr is handled correctly

done